src/main/java/com/springbeershop/springbeershop/entities/User.java

```diff
@@ -0,0 +1,53 @@
+package com.springbeershop.springbeershop.entities;
+
+import java.util.Date;
+
+import javax.persistence.Column;
+import javax.persistence.Entity;
+import javax.persistence.GeneratedValue;
+import javax.persistence.GenerationType;
+import javax.persistence.Id;
+import javax.persistence.PrimaryKeyJoinColumn;
+import javax.persistence.Table;
+
+import lombok.Data;
+
+
+@Data
+@Entity
+@Table(name = "user", schema = "public")
+public class User
+{
+    @Id
+    @GeneratedValue(strategy = GenerationType.IDENTITY)
+    @PrimaryKeyJoinColumn
```

Comment:
Co to w ogole za adnotacja?

Reply:
nie wiem czy to jest dobrze, ale napewno ta adnotacja przyda sie do mapowania z innymi zasobami takimi jak np ulubione piwa tego użytkownika

Reply:
No moze dobrze moze nie. Generalnie ta adnotacja nam  sie do niczego nie przyda. Ona jest uzywana w nieco innych przypadkach. Do wywalenia :D Przynajmniej na razie. nigdzie jej nie wykorzystujemy. Bedzie potrzebna (a zobaczysz ze nie bedzie xD) to dodamy.